lib/rest_connection/rightscale/server_interface.rb

```diff
@@ -229,38 +198,32 @@ def stop
   end
 
   def launch
-    return @impl.launch if @multicloud
-    return @impl.start unless @multicloud
+    return @impl.launch
   end
 
   def terminate
-    return @impl.terminate if @multicloud
-    return @impl.stop unless @multicloud
+    return @impl.terminate
   end
 
   def start_ebs
-    return connection.logger("WARNING: Gateway Servers do not support start_ebs. Ignoring.") if @multicloud
-    return @impl.start_ebs unless @multicloud
+    return connection.logger("WARNING: Gateway Servers do not support start_ebs. Ignoring.")
```

Comment:
Unless there is another method that we should be calling, but I don't remember seeing it.